src/content/chainlink-functions/tutorials/importing-packages.mdx

```diff
@@ -0,0 +1,222 @@
+---
+section: chainlinkFunctions
+date: Last Modified
+title: "Using Imports with Functions"
+metadata:
+  linkToWallet: true
+whatsnext:
+  {
+    "Try out the Chainlink Functions Tutorials": "/chainlink-functions/tutorials",
+    "Read the Architecture to understand how Chainlink Functions operates": "/chainlink-functions/resources/architecture",
+  }
+---
+
+import { Aside, CopyText } from "@components"
+import { Tabs } from "@components/Tabs"
+import ChainlinkFunctions from "@features/chainlink-functions/common/ChainlinkFunctions.astro"
+
+This tutorial demonstrates how to import modules and use them with your Functions source code. Modules that are imported into Functions source code must meet the following requirements:
+
+- Each import must be 10 MB or less in size.
+- Up to 20 imports total are supported.
```

Comment:
We actually increased this to 100 by default. Can you update this here (and anywhere else you refer to this limit).
The 10MB limit has remained the same though.

Reply:
Updated